examples/JavaProntoClient/build.gradle

```diff
@@ -0,0 +1,50 @@
+
+apply plugin: 'java'
+apply plugin: 'war' 
+apply plugin: 'com.bmuschko.tomcat'
+ 
+repositories {
+    jcenter()
+    
+    // Used for VANTIQ SDK
+    maven {
+    	url "https://dl.bintray.com/vantiq/maven"

```

Comment:
The indentation here looks inconsistent. Top-level looks to be 4 spaces, this looks like fewer. Can you make sure it's consistent throughout the file?